pkg/anago/anago.go

```diff
@@ -39,6 +39,10 @@ const (
 
 	// gitRoot is the local repository root of k/k.
 	gitRoot = workspaceDir + "/src/k8s.io/kubernetes"
+
+	// releaseNotesHTMLFile is the name of the release notes in HTML
+	releaseNotesHTMLFile = "/src/release-notes.html"
+	releaseNotesJSONFile = "/src/release-notes.json"
```

Comment:
Tiny nit: should we add a comment here for the sake of consistency?

Reply:
Done :)